libs/ledgerjs/packages/types-cryptoassets/src/index.ts

```diff
@@ -195,6 +195,7 @@ export type ExplorerView = {
   tx?: string;
   address?: string;
   token?: string;
+  stakePool?: string;
```

Comment:
the wording "stakePool" appear to be very specific to cardano. could it maybe be named "pool" instead?

Reply:
Sure @gre, we can do that. Also, it is very similar with other currency's validator I'll talk to Hakim and change it 👍